ocaml/xapi/xapi_globs.ml

```diff
@@ -894,7 +894,13 @@ let accept_sm_plugin name =
 
 let nvidia_multi_vgpu_enabled_driver_versions = ref ["430.19";"430.42";"440.00+"]
 let nvidia_default_host_driver_version = "0.0"
-let nvidia_t4_sriov = ref true
+
+type nvidia_t4_sriov =
+  | Nvidia_T4_SRIOV
+  | Nvidia_LEGACY
+  | Nvidia_DEFAULT
+
+let nvidia_t4_sriov = ref Nvidia_T4_SRIOV
```

Comment:
I assume this will get set to Nvidia_DEFAULT before merging, and just set to T4 on feature branch?
That is fine, another approach would be to change the default in `SOURCES/xenopsd/xenopsd-conf` on the feature branch.

Reply:
Yes, we will use `Nvidia_DEFAULT` (using the XML) before merging. This is just here to simplify development.